apps/products/models.py

```diff
@@ -187,48 +181,48 @@ def products_picture(instance, filename):
-    purchase_rate = models.DecimalField(max_digits=18, decimal_places=2)
-    purchase_rate_factor = models.DecimalField(max_digits=18, decimal_places=2)
-    purchase_discount = models.DecimalField(max_digits=18, decimal_places=2)
+    sales_description = models.TextField(null=True, default=None)
+    sales_gl_id = models.ForeignKey(ProductSalesGl, on_delete=models.CASCADE, db_column = 'sales_gl_id')
+    mrp = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    minimum_price = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    sales_rate = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    wholesale_rate = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    dealer_rate = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    rate_factor = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    discount = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    dis_amount = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    purchase_description = models.TextField(null=True, default=None)
+    purchase_gl_id = models.ForeignKey(ProductPurchaseGl, on_delete=models.CASCADE, db_column = 'purchase_gl_id')
+    purchase_rate = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    purchase_rate_factor = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
+    purchase_discount = models.DecimalField(max_digits=18, decimal_places=2, null=True, default=None)
     item_type_id = models.ForeignKey(ProductItemType, on_delete=models.CASCADE, null=True, default=None, db_column = 'item_type_id')
-    minimum_level = models.IntegerField()
-    maximum_level = models.IntegerField()
-    salt_composition = models.TextField()
+    minimum_level = models.IntegerField(null=True, default=None)
+    maximum_level = models.IntegerField(null=True, default=None)
+    salt_composition = models.TextField(null=True, default=None)
     drug_type_id = models.ForeignKey(ProductDrugTypes, on_delete=models.CASCADE, null=True, default=None, db_column = 'drug_type_id')
-    weighscale_mapping_code = models.CharField(max_length=50)
+    weighscale_mapping_code = models.CharField(max_length=50, null=True, default=None)
     brand_id = models.ForeignKey(ProductBrands, on_delete=models.CASCADE, null=True, default=None, db_column = 'brand_id')
-    purchase_warranty_months = models.IntegerField()
-    sales_warranty_months = models.IntegerField()
+    purchase_warranty_months = models.IntegerField(null=True, default=None)
+    sales_warranty_months = models.IntegerField(null=True, default=None)
+    STATUS_CHOICES = [
+        ('Active', 'Active'),
+        ('Inactive', 'Inactive'),
+        ('Pending','Pending'),
+    ]
```

Comment:
status ENUM('Active', 'Inactive'), correct it.

Reply:
changed